Assets/Scripts/Controllers/MouseController.cs

```diff
@@ -580,6 +570,7 @@ private void ShowFurnitureSpriteAtTile(string furnitureType, Tile t)
         sr.sprite = fsc.GetSpriteForFurniture(furnitureType);
 
         if (WorldController.Instance.World.IsFurniturePlacementValid(furnitureType, t) &&
+            WorldController.Instance.World.IsFurnitureWorkSpotClear(furnitureType, t) && 
```

Comment:
The work spot is part of the furniture, and the validity of the placement of the furniture also depends on that spot. So for me, Furniture placement should also check it.